client/src/components/AuditAdmin/AuditAdminView.test.tsx

```diff
@@ -23,43 +20,40 @@ import {
   jurisdictionErrorFile,
   standardizedContestsFile,
 } from './Setup/Participants/_mocks'
+import { queryClient } from '../../App'
 
 const getJurisdictionFileStatusMock = getJurisdictionFileStatus as jest.Mock
 const getRoundStatusMock = getRoundStatus as jest.Mock
 
-jest.mock('react-router-dom', () => ({
-  ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
-  useRouteMatch: jest.fn(),
-  useParams: jest.fn(),
-}))
-const paramsMock = useParams as jest.Mock
-paramsMock.mockReturnValue({
-  electionId: '1',
-  view: 'setup',
-})
-
 jest.mock('./useSetupMenuItems/getJurisdictionFileStatus')
 jest.mock('./useSetupMenuItems/getRoundStatus')
 getJurisdictionFileStatusMock.mockReturnValue('PROCESSED')
 getRoundStatusMock.mockReturnValue(false)
 
 jest.mock('axios')
 
-afterEach(() => {
-  paramsMock.mockReturnValue({
-    electionId: '1',
-    view: 'setup',
-  })
-})
+// AuditAdminView will only be rendered once the user is logged in, so
+// we simulate that.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const AuditAdminViewWithAuth = (props: any) => {
```

Comment:
Could we type this to indicate that we expect the provided props to be of type `RouteProps` and just cast to `any` when we pass to `AuditAdminView` (which I see [doesn't list any props](https://github.com/votingworks/arlo/blob/77785622733b49e705018ee4260566bcc4a97a14/client/src/components/AuditAdmin/AuditAdminView.tsx#L27))?